vllm/hpu/ops.py

```diff
@@ -38,8 +38,8 @@ def fetch_from_cache(cache, blocks, permutations):
     return [cache.index_select(0, blocks[:, i]).permute(permutations) for i in range(blocks.size(1))]
 
 
-@hpu_utils.with_mark_steps
-def paged_attention_v1(query, key_cache, value_cache, head_mapping, scale, block_tables, context_lens, block_size, alibi_slopes, kv_cache_dtype=None) -> None:
+def paged_attention_v1(query, key_cache, value_cache, head_mapping, scale, block_tables, context_lens, block_size, alibi_slopes, kv_cache_dtype=None,
```

Comment:
why is @hpu_utils.with_mark_steps removed here?

Reply:
we want that all convert to\from hf8 will be in the same graph
so we remove this mark step and add one outside the transformer block align to ohf version.